### PR TITLE
Remove passage

### PR DESCRIPTION
**Pull request by kramer65**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ def tryStep(String message, Closure block, Closure tearDown = null) {
         block();
     }
     catch (Throwable t) {
-        slackSend message: "${env.JOB_NAME}: ${message} failure ${env.BUILD_URL}", channel: '#iotsignals', color: 'danger'
+        slackSend message: "${env.JOB_NAME}: ${message} failure ${env.BUILD_URL}", channel: '#waarnemingen-deployments', color: 'danger'
```
Comment (jklardie) [inline Jenkinsfile]: use vars

File: Jenkinsfile [hunk 2]
```diff
@@ -29,8 +29,8 @@ node {
 
     stage("Build dockers") {
         tryStep "build", {
-            docker.withRegistry('https://repo.secure.amsterdam.nl','docker-registry') {
-                def api = docker.build("datapunt/iotsignals:${env.BUILD_NUMBER}", "api")
+            docker.withRegistry('https://repo.data.amsterdam.nl','docker-registry') {
+                def api = docker.build("datapunt/waarnemingen-mensen:${env.BUILD_NUMBER}", "api")
```
Comment (jklardie) [inline Jenkinsfile]: use vars

File: api/src/settings/settings.py [hunk 3]
```diff
@@ -31,17 +31,16 @@
 ALLOWED_HOSTS = ['*']
 
 # Application definition
-HEALTH_MODEL = 'passage.Passage'
+HEALTH_MODEL = 'peoplemeasurement.PeopleMeasurement'
```
Comment (jklardie) [inline api/src/settings/settings.py]: People of mensen?

File: Jenkinsfile (hunk 1 above)
Comment (kramer65) [reply Jenkinsfile]: Will implement better Jenkinsfile later

File: api/src/settings/settings.py (hunk 3 above)
Comment (kramer65) [reply api/src/settings/settings.py]: Noted. Will rename later.

**Pull request merged**